packages/govern-core/contracts/ERC3000Registry.sol

```diff
@@ -7,14 +7,20 @@ pragma solidity 0.6.8;
 import "erc3k/contracts/ERC3000Executor.sol";
 import "erc3k/contracts/ERC3000.sol";
 
+
+// Note: should this be part of the standard?
+// Why not leveraging ENS for this?
```

Comment:
IMO they are different things and this is just so lightweight as the default. The Factory could also assign you an ENS name, but tbh I don't like how we do in aOS in which an ENS name is almost a dependency